src/System.Runtime/tests/System/String.cs

```diff
@@ -1522,28 +1460,40 @@ public static void TestToLowerToUpperInvariant_ASCII()
         Assert.Equal(asciiUpper, ascii.ToUpperInvariant());
     }
 
-    [Fact]
-    public static void TestTrim()
+    [Theory]
+    [InlineData("  Hello  ", ' ', "Hello")]
+    [InlineData(".  Hello  ..", '.', "  Hello  ")]
```

Comment:
Whether as part of this PR or subsequently, we should add tests that cover supplying multiple chars to Trim/TrimStart/TrimEnd.